doc/source/getting_started/docker/windows_container.rst

```diff
@@ -0,0 +1,165 @@
+
+Build or install the Geometry Service image
+-------------------------------------------
+
+There are two options for users to install the PyAnsys Geometry image:
+
+* Downloading it from the :ref:`GitHub Container Registry <ref_windows_docker_ghcr>`.
+* :ref:`Building the Geometry Service Windows container <ref_windows_docker_fromscratch>`.
+
+.. _ref_windows_docker_ghcr:
+
+GitHub Container Registry
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. note::
+
+    This option is only available for users with write access to the repository or
+    part of the Ansys organization.
+
+Once Docker is installed on your machine, follow these steps to download the Windows Docker
+container for the Geometry service and install this image.
+
+#. Using your GitHub credentials, download the Docker image from the `PyAnsys Geometry repository <https://github.com/ansys/pyansys-geometry>`_
+   on GitHub.
+
+#. Use a GitHub personal access token with permission for reading packages to authorize Docker
+   to access this repository. For more information, see `Managing your personal access tokens
+   <https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens>`_
+   in the GitHub documentation.
+
+#. Save the token to a file with this command:
+
+   .. code-block:: bash
+
+       echo XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX > GH_TOKEN.txt
+
+#. Authorize Docker to access the repository. To see the commands to run, click the tab for your OS.
+
+   .. tab-set::
+
+       .. tab-item:: Powershell
+
+           .. code-block:: pwsh
+
+               $env:GH_USERNAME=<my-github-username>
+               cat GH_TOKEN.txt | docker login ghcr.io -u $env:GH_USERNAME --password-stdin
+
+       .. tab-item:: Windows CMD
+
+           .. code-block:: bash
+
+               SET GH_USERNAME=<my-github-username>
+               type GH_TOKEN.txt | docker login ghcr.io -u %GH_USERNAME% --password-stdin
+
+
+#. Pull the Geometry service locally using Docker with a command like this:
+
+   .. code:: bash
+
+      docker pull ghcr.io/ansys/geometry:windows-latest
+
+.. _ref_windows_docker_fromscratch:
+
+Building the Geometry Service Windows container
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Geometry service Docker containers can be easily built by following
+these steps.
+
+Inside the repository's ``docker`` folder, there are two ``Dockerfile`` files:
+
+* ``Dockerfile.linux``: this file builds the Linux-based Docker image.
+* ``Dockerfile.windows``: this file builds the Windows-based Docker image.
+
+Depending on the characteristics of the Docker engine installed in your
+machine, either one or the other has to be built.
+
+In this guide, focus is set on building the ``Dockerfile.windows`` image.
+
+Prerequisites
+~~~~~~~~~~~~~
+
+* Ensure that ``docker`` is installed in your machine.
+  If you do not have ``docker`` available, please refer to
+  :ref:`Running Windows Docker containers <ref_running_windows_containers>`.
```

Comment:
```suggestion
* Ensure that ``docker`` is installed in your machine.
  If you do not have ``docker`` available, see
  :ref:`Running Windows Docker containers <ref_running_windows_containers>`.
```
Same qujestion about using Docker (proper noun) rather than tagging as a literal.